operator/src/main/resources/scripts/modelInImage.sh

```diff
@@ -384,6 +389,61 @@ function checkDirNotExistsOrEmpty() {
   trace "Exiting checkDirNotExistsOrEmpty"
 }
 
+# limit the file extensions in the model directories
+
+function checkModelDirectoryExtensions() {
+  trace "Entering checkModelDirectoryExtensions"
+
+  cd ${IMG_MODELS_HOME}
+  counter=$(ls  -I  "*.yaml" -I "*.zip" -I "*.properties" | wc -l)
+  if [ $counter -ne 0 ] ; then
+    trace SEVERE "Model image directory ${IMG_MODELS_HOME} contains files with unsupported extensions. " \
+      "Expected extensions: .yaml, .properties, or .zip"
+    trace SEVERE "Model image directory files with unsupported extensions: " \
+      "'$(ls -I "*.yaml" -I "*.zip" -I "*.properties")'"
+    exitOrLoop
+  fi
+  if [ -d ${WDT_CONFIGMAP_ROOT} ] ; then
+    cd ${WDT_CONFIGMAP_ROOT}
+    counter=$(ls  -I  "*.yaml" -I "*.properties" | wc -l)
```

Comment:
Would it be OK to secretly allow a '.zip'?  A 'small' app could fit in the cm.  Or do we specifically want to disallow this?

Reply:
Lets's disallow for now, There are quite a few places to change and it's hard to tell user to try it and if it doesn't fit do it other way.  This PR is for checking extension and WDT version, not dealing with features.

Reply:
Sure. Thanks for considering it. Resolving.